orunmila.go

```diff
@@ -233,11 +230,14 @@ func main() {
 
 	// check if db file exists
 
-	file, err := os.Open(*dbPtr)
-	file.Close()
-	if errors.Is(err, os.ErrNotExist) {
-		log.Debugln("database does not exist, creating...")
-		createDB(*dbPtr)
+	_, err := os.Stat(*dbPtr)
```

Comment:
the reason i did open and not stat here was that stat will **not** fail on directories but Open will fail

Reply:
You're right, will add a check if it's a directory or not, the reason why i stick with `Stat()` is that (probably fast since it uses less syscalls?) and it doesn't require resources as `Open()` does